ceph_iscsi_config/settings.py

```diff
@@ -118,6 +120,14 @@ def normalize(k, v):
                 "logger_level": logging.DEBUG
                 }
 
+    sync_required = ["cluster_name",
```

Comment:
Perhaps this should instead be a blacklist for excluding specific configuration options that don't matter if they differ between gateways. Right now I think this list is missing lots of important things that shouldn't be allowed to diverge: i.e. api_ssl_verify, api_user, api_password, .....

Reply:
Fixed. I'm now using a blacklist of setting that should be excluded from the "hash" calculation.